scapy/contrib/aoe.py

```diff
@@ -0,0 +1,139 @@
+#! /usr/bin/env python
+
+# Copyright (C) 2018 antoine.torre <torreantoine1@gmail.com>
+##
+# This program is published under a GPLv2 license
+
+
+# scapy.contrib.description = ATA Over Internet
+# scapy.contrib.status = loads
+
+from scapy.packet import Packet, bind_layers
+from scapy.fields import FlagsField, XByteField, ByteField, XShortField, \
+    ShortField, StrLenField, BitField, BitEnumField, ByteEnumField, \
+    FieldLenField, PacketListField, FieldListField, MACField, PacketField, \
+    ConditionalField, XIntField
+from scapy.layers.l2 import Ether
+from scapy.data import ETHER_ANY
+
+
+class IssueATACommand(Packet):
+    name = "Issue ATA Command"
+    fields_desc = [FlagsField("flags", 0, 8, "zezdzzaw"),
+                   XByteField("err_feature", 0),
+                   ByteField("sector_count", 1),
+                   XByteField("cmd_status", 0xec),
+                   XByteField("lba0", 0),
+                   XByteField("lba1", 0),
+                   XByteField("lba2", 0),
+                   XByteField("lba3", 0),
+                   XByteField("lba4", 0),
+                   XByteField("lba5", 0),
+                   XShortField("reserved", 0),
+                   StrLenField("data", "",
+                               length_from=lambda x: x.sector_count * 512)]
+
+    def extract_padding(self, p):
```

Comment:
As spotted by Codacy, this should be `    def extract_padding(self, s):` to keep the same parameter names than in the parent method.